sr/comp/mixtape/cli.py

```diff
@@ -202,6 +202,9 @@ def main():
         parser.print_help()
         return
 
+    if os.path.isfile(args.mixtape_directory):
+        exit("File entered for the playlist when directory expected!")
```

Comment:
A potential improvement here might be to include the name of the file something like:
```suggestion
        exit(f"{args.mixtape_directory!r} is a file. You must provide the directory containing the configuration and audio files.")
```